src/__tests__/http-test.ts

```diff
@@ -1117,9 +1116,11 @@ function runTests(server: Server) {
     app.get(
       urlString(),
       graphqlHTTP((req, res) => {
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
         if (req) {
           hasRequest = true;
         }
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
add fixme comment to not that we want to fix it in the future.

Reply:
Also it strange because `istanbul` has coverage for those lines.

Reply:
@IvanGoncharov Fixed in a separate PR